docs/src/tutorial/cython_tutorial.rst

```diff
@@ -165,6 +221,72 @@ which we can try out in the interactive interpreter as follows::
 See, it works! And if you're curious about how much work Cython has saved you,
 take a look at the C code generated for this module.
 
+
+It is always good to check where is the Python interaction in the code with the
```

Comment:
-> "Cython has a way to visualise where interaction with Python objects and Python's C-API is taking place. For this, pass the `annotate=True` parameter to `cythonize()`:"